python/doc/conf.py.in

```diff
@@ -189,7 +189,7 @@ pygments_style = 'friendly'
 
 # If true, keep warnings as "system message" paragraphs in the built documents.
 #keep_warnings = False
-
+suppress_warnings=["config.cache"]
```

Comment:
this is caused by sphinx-gallery not having caught up yet to sphinx 7.3:
https://github.com/sphinx-gallery/sphinx-gallery/issues/1286

but these warnings can be useful
I suggest to drop this commit and downgrade sphinx to 7.2.x until sphinx gallery releases a new version:
`pip install "sphinx<7.3"`

Reply:
ok done